docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/view_Figure_Position.png" width="100%">
+</td>
+</tr>
+</table>
+
+`Orientation` of `View` is the rotation from its default orientation and `Scale` of `View` is the size ratio between the size to be rendered and the default size. For these two transforms, `View` is rotated and scaled around [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint). NUI also provides predefined presets of `PivotPoint` identical to `ParentOrigin`. Alternatively, you can use custom values.
+
+`Orientation` and `Scale` of `View` is also affected by the transforms of its parent. In the final calculation of `View`, `Orientation` and `Scale` of parents are first applied to the child `View` with parent's `PivotPoint`, and then `Orientation` and `Scale` of child are applied.
+
+The following examples demonstrate the rotation and scaling that change `Orientation` and `Scale` respectively. `View` *A* is a child of `View` *B*. Therefore, transforms of *A* only affects *A*. however, the transforms of parent *B* influences its child *A*. The black dot in the figure is `PivotPoint` that is used by the transform.
+
+**Figure: Rotation and Scaling.**
+
+<table style="width:100%">
+<tr>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Orientation.gif" width="100%">
+</td>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Scale.gif" width="100%">
+</td>
+</tr>
+</table>
 
-```
-TextLabel label = new TextLabel( "Hello World" );
-label.BackgroundColor = Color.Red;
-```
 
-**Figure: TextLabel object with a red background**
+<a name="2"></a>
+## Directional Navigation
+
+NUI also provides directional navigation between `View` using arrow keys on the keyboard. You can simply set the following properties to specify the next `FocusableViews` for each direction:
+
+- [View.UpFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_UpFocusableView)
+
+- [View.DownFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_DownFocusableView)
+
+- [View.LeftFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_LeftFocusableView)
+
+- [View.RightFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_RightFocusableView)
+
+When an arrow key is pressed and the current `View` has next `FocusableView` for the direction, the system passes focus to the next `FocusableView`. However, if the current `View` does not have the next `FocusableView` for the direction or the next `ForcusableView` is not on the window, the property returns `NULL` and no change occurs.
```

Comment:
for the direction, 

Question. How about "for that direction, "?

Reply:
@mijin85cho 
I  agree with you.

Reply:
Done, Thank you.